frame/contracts/src/rent.rs

```diff
@@ -233,14 +236,11 @@ where
 		alive_contract_info: AliveContractInfo<T>,
 		current_block_number: T::BlockNumber,
 		verdict: Verdict<T>,
-		allow_eviction: bool,
-	) -> Result<Option<ContractInfo<T>>, DispatchError> {
-		match verdict {
-			Verdict::Exempt => return Ok(Some(ContractInfo::Alive(alive_contract_info))),
-			Verdict::Evict { amount: _ } if !allow_eviction => {
-				Ok(None)
-			}
-			Verdict::Evict { amount } => {
+		evictable_code: Option<PrefabWasmModule<T>>,
```

Comment:
Maybe you can precise in the doc that this means the verdict is allowed to evict the contract

Reply:
Done.